test/services/claimToAdjust/ClaimToAdjustPoaCalculationServiceSpec.scala

```diff
@@ -16,8 +16,71 @@
 
 package services.claimToAdjust
 
-class ClaimToAdjustPoaCalculationServiceSpec {
+import mocks.connectors.MockClaimToAdjustPoaConnector
+import models.claimToAdjustPoa.ClaimToAdjustPoaResponse.{ClaimToAdjustPoaError, ClaimToAdjustPoaInvalidJson, ClaimToAdjustPoaSuccess, UnexpectedError}
+import models.claimToAdjustPoa.{ClaimToAdjustPoaRequest, MainIncomeLower}
+import models.incomeSourceDetails.TaxYear
+import services.ClaimToAdjustPoaCalculationService
+import testConstants.BaseTestConstants.{testNino, testTaxYear}
+import testUtils.TestSupport
 
-  // to be implemented
+class ClaimToAdjustPoaCalculationServiceSpec extends TestSupport with MockClaimToAdjustPoaConnector{
```

Comment:
:thinking: - use of whenReady is fine, but preference would be futureValue, there are quite a few examples we have in out code base: 
https://www.scalatest.org/scaladoc/3.0.6/org/scalatest/concurrent/ScalaFutures.html

Reply:
Done, will raise another small PR for it